### PR TITLE
Feature/add settings home page

### PR DESCRIPTION
**Pull request by CDRussell**
Asana Issue URL: https://app.asana.com/0/414730916066338/513409707430058

## Description
Add a way to get to the settings page from the home screen.

## Steps to Test this PR:
1. Launch the app
1. ensure there is a transparent toolbar on the home page
1. ensure there is an overflow menu, with an option to view "Settings"
1. ensure clicking on that takes you to settings
1. finally, navigate to settings, and then onto the about screen. from there, click the "more" hyperlink. Ensure you are shown the browser activity with the correct url, and that you can navigate "back" from this screen

### CODE REVIEW

**Comment by CDRussell** [comment]
device-2018-01-08-182035

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -20,5 +20,6 @@
     <color name="orange">#de5833</color>
     <color name="powderBlue">#50BFFF</color>
     <color name="midGreen">#3fa140</color>
+    <color name="transparent">#00000000</color>
```
Comment (subsymbolic) [inline app/src/main/res/values/colors.xml]: Is this used?

File: app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt [hunk 2]
```diff
@@ -68,8 +68,10 @@ class SettingsActivity : DuckDuckGoActivity() {
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         when(requestCode) {
             REQUEST_CODE_ABOUT_DDG -> {
-                setResult(AboutDuckDuckGoActivity.RESULT_CODE_LOAD_ABOUT_DDG_WEB_PAGE)
-                finish()
+                if(resultCode == AboutDuckDuckGoActivity.RESULT_CODE_LOAD_ABOUT_DDG_WEB_PAGE ) {
```
Comment (subsymbolic) [inline app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt]: Not sure the when / if combination works well here. Can we simplify this into something along the lines of:
`if (requestCode == REQUEST_CODE_ABOUT_DDG && resultCode == AboutDuckDuckGoActivity.RESULT_CODE_LOAD_ABOUT_DDG_WEB_PAGE )`

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (CDRussell) [reply app/src/main/res/values/colors.xml]: This was used while i was experimenting with getting the toolbar to be transparent. Not any more; good spot.

**Comment by CDRussell** [comment]
_Feedback addressed; merging_

**Pull request merged**